packages/dd-trace/src/llmobs/telemetry.js

```diff
@@ -18,6 +18,27 @@ const LLMObsTagger = require('./tagger')
 
 const llmobsMetrics = telemetryMetrics.manager.namespace('mlobs')
 
+function extractIntegrationFromTags (tags) {
+  if (!tags || !Array.isArray(tags)) return null
```

Comment:
```suggestion
  if (!Array.isArray(tags)) return null
```
nit: `Array.isArray` checks nullish